creds/views/web.py

```diff
@@ -2,6 +2,30 @@
 from creds.forms import (
     CreateUserForm, UserPassResetForm, UserLoginForm
 )
+from django.views import View
+from django.contrib.auth.models import User
+
+
+class UserSignUp(View):
+    """
+    View to handle user sign up
+    """
+    def get(self, request):
+        context = {'form': CreateUserForm()}
+        return render(request, 'creds/signup.html', context)
+
+    def post(self, request):
+        form = CreateUserForm(request.POST)
+        if not form.is_valid():
+            context = {'form': CreateUserForm(), 'sign_up_errors': form.errors}
+            return render(request, 'creds/signup.html', context)
+
+        # create use if form is valid
```

Comment:
I think the comment is irrelevant and we can do away with it given the code is crystal clear.